frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -168,13 +168,37 @@
   "An error has occurred": "An error has occurred",
   "Create StorageSystem": "Create StorageSystem",
   "StorageSystem is an entity of OpenShift Data Foundation. It represents all of the required storage and compute resources.": "StorageSystem is an entity of OpenShift Data Foundation. It represents all of the required storage and compute resources.",
+  "Details": "Details",
+  "Replicas": "Replicas",
+  "Inventory": "Inventory",
   "Not available": "Not available",
+  "Image states info": "Image states info",
+  "<0><0><0> What does each state mean?</0></0><1><0>Starting replay:</0> Initiating image (PV) replication process.</1><2><0>Replaying:</0> Image (PV) replication is ongoing or idle between clusters.</2><3><0>Stopping replay:</0> Image (PV) replication process is shutting down.</3><4><0>Stopped:</0> Image (PV) replication process has shut down.</4><5><0>Error:</0> Image (PV) replication process stopped due to an error. </5><6><0>Unknown:</0> Unable to determine image (PV) state due to an error. Check your network connection and remote cluster mirroring daemon.</6></0>": "<0><0><0> What does each state mean?</0></0><1><0>Starting replay:</0> Initiating image (PV) replication process.</1><2><0>Replaying:</0> Image (PV) replication is ongoing or idle between clusters.</2><3><0>Stopping replay:</0> Image (PV) replication process is shutting down.</3><4><0>Stopped:</0> Image (PV) replication process has shut down.</4><5><0>Error:</0> Image (PV) replication process stopped due to an error. </5><6><0>Unknown:</0> Unable to determine image (PV) state due to an error. Check your network connection and remote cluster mirroring daemon.</6></0>",
```

Comment:
Seems like we did not translate it properly.
We should translate in sentence or paragraphs, so it can be translated easily by tranlsators